xml/System.Runtime.CompilerServices/Unsafe.xml

```diff
@@ -49,11 +49,13 @@
         <Parameter Name="elementOffset" Type="System.Int32" Index="1" FrameworkAlternate="dotnet-plat-ext-2.1;dotnet-plat-ext-2.2;netcore-3.0" />
       </Parameters>
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <param name="source">To be added.</param>
-        <param name="elementOffset">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="T">The type of void pointer.</typeparam>
+        <param name="source">The void pointer to add the offset to.</param>
+        <param name="elementOffset">The offset to add.</param>
+        <summary>
+          Adds an element offset to the given void pointer.
```

Comment:
nit: Can you put the summary in a single line with its tags?

Reply:
I've added a suggestion for that